crates/ruff_linter/src/importer/insertion.rs

```diff
@@ -63,9 +63,10 @@ impl<'a> Insertion<'a> {
                 return Insertion::inline(" ", location.add(offset).add(TextSize::of(';')), ";");
             }
 
-            // If the first token after the docstring is a continuation character (i.e. "\"), advance
-            // an additional row to prevent inserting in the same logical line.
-            if match_continuation(locator.after(location)).is_some() {
+            // While the first token after the docstring is a continuation character (i.e. "\"), advance
+            // additional rows to prevent inserting in the same logical line. Continue advancing
+            // while subsequent lines also contain a continuation.
```

Comment:
I think this part is basically redundant now.

```suggestion
            // additional rows to prevent inserting in the same logical line.
```